server/src/app.controller.ts

```diff
@@ -29,11 +30,15 @@ export class AppController {
       });
     } catch (e) {
       if (e instanceof HttpException) {
-        res.status(401).send({ errors: [e] });
+        throw e;
       } else {
-        console.log(e);
-
-        res.status(500).send({ errors: [e] });
+        const error = {
+          code: "LOGIN_ERROR",
+          title: "Failed to login",
+          detail: "An unknown error occured while authenticating user",
```

Comment:
So this is like the "catch all" net that catches anything that slipped through the service correct?

Reply:
yep, exactly what I was thinking!